nxdrive/client/remote_client.py

```diff
@@ -92,11 +95,24 @@ def __init__(
             cert=cert,
         )
 
+        installation_type = "system-wide" if Options.system_wide else "user-only"
```

Comment:
Let's send less data for each call:
```suggestion
        installation_type = "system" if Options.system_wide else "user"
```